blocks/library/search/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+ import './editor.scss';
+import { registerBlockType } from '../../api';
+import InspectorControls from '../../inspector-controls';
+import TextControl from '../../inspector-controls/text-control';
+
+class SearchBlock extends Component {
```

Comment:
`SearchBlock` component should be moved to its own `edit.js` file so it could be overridden by the mobile app when necessary.